lib/Models/Videos.php

```diff
@@ -954,8 +954,12 @@ public static function parseEvent($eventType, $episode, $video)
             $livestream_link       = '';
 
             foreach ((array) $episode->publications[0]->attachments as $attachment) {
-                if ($attachment->flavor === "presenter/search+preview" || $attachment->type === "presenter/search+preview") {
-                    $preview = $attachment->url;
+                if ($attachment->flavor === "presenter/search+preview" || $attachment->type === "presenter/search+preview"
+                    ||  $attachment->type === "presenter/player+preview")
```

Comment:
```suggestion
                    || $attachment->flavor  === "presenter/player+preview" ||  $attachment->type === "presenter/player+preview")
```
In our opencast, the presenter preview is located in the flavor attribute.